glide.lock

```diff
@@ -221,9 +222,10 @@ imports:
   - jws
   - jwt
 - name: golang.org/x/sys
-  version: 7de4796419dc3b554e6dab3a119a62469569d299
+  version: 8f0908ab3b2457e2e15403d3697c9ef5cb4b57a9
```

Comment:
note: this version is older than before.

Reply:
Not entirely sure how that happened. I updated glide.yaml and regenerated the lock file. Could this be a problem?

Reply:
It's defined [in client-go's Godeps.json](https://github.com/kubernetes/client-go/blob/release-3.0/Godeps/Godeps.json#L219). Not sure how we get the other version.